### PR TITLE
KDE Frameworks: update to 5.29.0

### PR DESCRIPTION
**Pull request by dutsky**

### CODE REVIEW

File: srcpkgs/attica-qt5/template [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+revision=1
 wrksrc=${pkgname%-*}-${version}
 build_style=cmake
 configure_args="-DBUILD_TESTING=OFF"
 hostmakedepends="extra-cmake-modules qt5-host-tools qt5-devel"
 makedepends="qt5-devel"
 short_desc="A Qt library that implements the Open Collaboration Services API"
 maintainer="Juan RP <xtraeme@gmail.com>"
-license="GPL-3"
+license="LGPL-2.1"
```
Comment (Vaelatern) [inline srcpkgs/attica-qt5/template]: :+1:

File: srcpkgs/kdelibs4support/template [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 pkgname=kdelibs4support
-version=5.26.0
+version=5.28.0
 revision=1
 build_style=cmake
 configure_args="-DBUILD_TESTING=OFF"
-hostmakedepends="extra-cmake-modules perl-URI"
-makedepends="kded-devel kemoticons-devel kitemmodels-devel kparts-devel kunitconversion-devel kdesignerplugin-devel NetworkManager-devel kdoctools-devel libSM-devel"
+hostmakedepends="extra-cmake-modules perl-URI pkg-config"
+makedepends="kded-devel kemoticons-devel kitemmodels-devel kparts-devel kunitconversion-devel kdesignerplugin-devel networkmanager-qt5-devel kdoctools-devel libSM-devel"
```
Comment (Vaelatern) [inline srcpkgs/kdelibs4support/template]: :+1:

**Comment by dutsky** [comment]
There are 5.29.0 already available. Must I update it in this PR?

**Comment by Vaelatern** [comment]
Is `5.29.0` updated for all of these packages?

**Comment by dutsky** [comment]
Yes

**Comment by Vaelatern** [comment]
Then, if you don't mind, updates would be great.

**Comment by ihavesomepoblem** [comment]
Can we already run kde plasma?

**Comment by dutsky** [comment]
Yes you can, but only if you will build it from sources.
You can do this by typing this commands(assume you have ~/devel):
```
cd ~/devel
git clone https://github.com/dutsky/void-packages.git
cd void-packages
git checkout plasma
./xbps-pkg binary-bootstrap
./xbps-pkg pkg kde5 (it will take a **LOT** of time)
xbps-install --repository=hostdir/binpkgs/plasma kde5
sudo ln -s /etc/sv/dhcpd /var/service
sudo ln -s /etc/sv/sddm /var/service
```
and reboot, if sddm didn't started

**Pull request merged**